lib/api/client-commands.js

```diff
@@ -399,6 +399,97 @@ module.exports = function(client) {
       }
 
       return this.execute(function(u,i) {/* jshint browser:true */return (function(d){var e=d.createElement('script');var m=d.getElementsByTagName('head')[0];e.src=u;if(i){e.id=i;}m.appendChild(e);return e;})(document);}, args, callback);
-    }
+    },
+	
+    /**
+     * Gets the available log types
+     *
+     * ```
+     * this.demoTest = function(client) {
+     *   this.getLogTypes(function( typesArray ) {
+     *     
+     *   });
+     * };
+     * ```
+     *
+     * @method getLogTypes
+     * @param {function} [callback] Optional callback function to be called when the command finishes.
+     * @api commands
+	 * @see logTypes
+     */
+	getLogTypes: function( callback ) {
```

Comment:
The spacing and indentation here are a bit off. Could you please adhere to the existing coding style?